canvas/fizz-27-py/src/lib/courses.py

```diff
@@ -0,0 +1,33 @@
+from lib.canvasRequest import get
+import os
+from datetime import datetime
+
+def getCoursesByStatus(status):
```

Comment:
https://www.python.org/dev/peps/pep-0008/#id45:

> "Function names should be lowercase, with words separated by underscores as necessary to improve readability.
>
> Variable names follow the same convention as function names.
>
> mixedCase is allowed only in contexts where that's already the prevailing style (e.g. threading.py), to retain backwards compatibility."

Reply:
I didn't even notice this, I'm so much more use to non-snake case function names!  FYI, PyLint gives warnings about this.